frontend/src/styles/element-dark.scss

```diff
@@ -96,6 +96,7 @@ html.dark {
     --panel-terminal-tag-active-text-color: var(--panel-color-primary);
     --panel-terminal-tag-hover-text-color: var(--panel-color-primary);
     --panel-logs-bg-color: var(--panel-main-bg-color-9);
+    --panel-alert-bg-color: var(--panel-main-bg-color-10);
 
     --el-menu-item-bg-color: var(--panel-main-bg-color-10);
     --el-menu-item-bg-color-active: var(--panel-main-bg-color-8);
```

Comment:
No irregularities were found in the provided code changes. The addition of `--panel-alert-bg-color` suggests that alerts may include custom styling with this background color, enhancing user experience based on its usage within your application.